sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -649,6 +649,14 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val SHOW_STATS_IN_EXPLAIN =
```

Comment:
OK. But since it influences user interface, let's double check with others. @rxin @hvanhovell @cloud-fan Shall we show stats of LogicalPlan directly in explain command ?